core/src/migrations/000090-renamePropertyDirectlyMappedToIsPrimaryKey.ts

```diff
@@ -0,0 +1,20 @@
+import Sequelize from "sequelize";
```

Comment:
Note on Coordination - looks like https://github.com/grouparoo/grouparoo/pull/2527 has migration 90 so this should be 91